package.json

```diff
@@ -5,22 +5,25 @@
   "dependencies": {
     "@airyhq/components": "latest",
     "@stomp/stompjs": "^6.0.0",
+    "@types/facebook-js-sdk": "^3.3.0",
     "@types/node": "12.11.1",
-    "@types/react": "16.9.34",
     "@types/react-dom": "16.9.2",
     "@types/react-redux": "7.1.3",
     "@types/react-router-dom": "^5.1.0",
+    "@types/react": "16.9.34",
     "core-js": "3",
     "emoji-mart": "^3.0.0",
     "linkifyjs": "^2.1.9",
-    "preact": "^10.5.7",
+    "lodash-es": "4.17.15",
     "preact-router": "^3.2.1",
-    "react": "16.12.0",
+    "preact": "^10.5.7",
```

Comment:
Would be great (but optional) if you could fix this version

Reply:
We talked about it. This is just here because it was resorted. Leaving it as is.